### PR TITLE
benetech/ServiceNet#1200: Add possibility to check in beneficiary

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: src/main/webapp/app/modules/provider/provider-record.reducer.ts [hunk 1]
```diff
@@ -69,6 +77,11 @@ export default (state: ProviderRecordsState = initialState, action): ProviderRec
         updateSuccess: false,
         errorMessage: action.payload
       };
+    case FAILURE(ACTION_TYPES.CHECK_IN):
+      return {
+        ...state,
+        referralSent: false
```
Comment (jkondrat) [inline src/main/webapp/app/modules/provider/provider-record.reducer.ts]: can we rename it to 'checkedIn' or 'beneficiaryCheckedIn'? The name will clash with sending actual referrals

**Review by jkondrat — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/modules/provider/provider-record.reducer.ts (hunk 1 above)
Comment (awalkowiak) [reply src/main/webapp/app/modules/provider/provider-record.reducer.ts]: Fair point, will do.

File: src/main/webapp/i18n/en/referral.json [hunk 2]
```diff
@@ -19,11 +19,15 @@
   },
   "referral": {
     "labels": {
+      "close": "Close",
+      "homePage": "Go to Home Page",
       "or": "OR",
-      "send": "Send"
+      "send": "Send",
+      "sent": "Congratulations, referral sent!"
```
Comment (jkondrat) [inline src/main/webapp/i18n/en/referral.json]: this is misleading as we have another tab for sending referrals, I think it should say 'Beneficiary checked in successfully'

**Review by jkondrat — CHANGES_REQUESTED** [review]

File: src/main/webapp/i18n/en/referral.json (hunk 2 above)
Comment (awalkowiak) [reply src/main/webapp/i18n/en/referral.json]: Fair point, will do.

**Pull request merged**